app/models/container_project.rb

```diff
@@ -2,6 +2,7 @@ class ContainerProject < ActiveRecord::Base
   include CustomAttributeMixin
   include ReportableMixin
   belongs_to :ext_management_system, :foreign_key => "ems_id"
+  has_one  :container_resource_quota, :dependent => :destroy
```

Comment:
For as much as I know you can have multiple quotas per-namespace. Is that correct @deads2k @smarterclayton ?